lib/src/nexus_plot.cpp

```diff
@@ -34,6 +34,7 @@
 #include <winsock2.h>
 #endif
 
+#include <ert/ecl/ecl_sum.h>
```

Comment:
why added?

Reply:
That header is no longer included in nexus_plot.hpp, so it is needed here now.

Reply:
```
nex/python/nex/nex.cpp:1:31: fatal error: pybind11/pybind11.h: No such file or directory
 #include <pybind11/pybind11.h>
                               ^
compilation terminated.
error: command 'gcc' failed with exit status 1
python/CMakeFiles/nex-python.dir/build.make:57: recipe for target 'python/CMakeFiles/nex-python' failed
make[2]: *** [python/CMakeFiles/nex-python] Error 1
CMakeFiles/Makefile2:1247: recipe for target 'python/CMakeFiles/nex-python.dir/all' failed
make[1]: *** [python/CMakeFiles/nex-python.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```

Reply:
Should `<pybind11/pybind11.h>` be present?   If so, why do Travis pass?

Reply:
You need to install pybind11 with pip, or install requirements with pip

Reply:
Which argument to `cmake` should tell `cmake` where to find the header file?